packages/venia-ui/lib/components/CartPage/PriceAdjustments/__tests__/priceAdjustments.spec.js

```diff
@@ -3,7 +3,7 @@ import { createTestInstance } from '@magento/peregrine';
 
 import PriceAdjustments from '../priceAdjustments';
 
-jest.mock('../CouponCode/couponCode', () => 'CouponCode');
+jest.mock('../ShippingMethods/shippingMethods', () => 'ShippingMethods');
```

Comment:
Does `CouponCode` not need to be mocked also?

Reply:
Since I removed the `isOpen={true}` prop, its no longer rendered by default, so no need to mock.